README.md

```diff
@@ -200,17 +189,29 @@ make
 
 Now that the arkouda_server is built and tested, install the Python library
 
-## Installing the Arkouda Python Library
+## Installing the Arkouda Python Library and Dependencies
 
-```
+The Arkouda Python library along with it's dependent libraries are installed with pip. There are four types of 
+Python dependencies for the Arkouda developer to install: requires, dev, test, and doc. The required libraries, 
+which are the runtime dependencies of the Arkouda python library, are installed as follows:
+
+```bash
  pip3 install -e .
 ```
 
+Arkouda and the Python libaries required for development, test, and doc generation activities are installed
+as follows:
+
+```bash
+pip3 install -e .[dev]
+```
+
 ## Testing Arkouda
 
-There are two unit test suites for Arkouda, one for Python and one for Chapel. As mentioned above, the Arkouda Python 
-test harness leverages the [pytest](https://docs.pytest.org/en/latest/) and [pytest-env](https://pypi.org/project/pytest-env/) 
-libraries, whereas the Chapel test harness does not require any external librares.
+There are two unit test suites for Arkouda, one for Python and one for Chapel. As mentioned above, the Arkouda  
+Python test harness leverages multiple libraries such as [pytest](https://docs.pytest.org/en/latest/) and 
+[pytest-env](https://pypi.org/project/pytest-env/) that must be installed via `pip3 install -e .[test]`, 
```

Comment:
`[test]` -> `[dev]`?

Reply:
@ronawho yes, we folded doc and test dependencies into dev

Reply:
Right, I was wondering if this one needs to be updated to reflect that too.